src/oppia.yaml

```diff
@@ -0,0 +1,34 @@
+---
+description: >
+  Oppia is an online leanring tool that enables anyone to create and share
+  interactive activiies.
+  
+  In addition the core team also maintains a set of free and effective lessions
+  that are targeted at learners who lack access to educational resources.
+license: Apache-2.0
+name: Oppia
+programmingLanguages:
+  - python
+  - typescript
+rating: 5
+repoUrl: "https://github.com/oppia/oppia"
+sdgs: [4, 10]
+websiteUrl: "https://www.oppia.org"
+
+contributionGuidelinesUrl: "https://github.com/oppia/oppia/wiki"
+logoUrl: "https://avatars0.githubusercontent.com/u/11620230?s=200&v=4"
+starsUrl: "https://img.shields.io/github/stars/oppia/oppia.svg\
+           ?style=social&label=Star&maxAge=2592000"
+numberContributors:
+  url: "https://api.github.com/repos/oppia/oppia/stats/contributors"
```

Comment:
Since we can automate this part for github, we don't really need to add it :)

Reply:
The question is at which point should it be automatically completed? When creating the YAML or when reading the project files? :thinking:

I think I'll leave it for now, since all of our projects up until now have it. If we decide to change it, we can just go over all of the projects rather easily.

Reply:
if following the logic "The json file is the unique source of truth", then it should be parsed by the json generator :).